src/vcf/vcf_v41.ragel

```diff
@@ -226,7 +226,8 @@
     # Fileformat line
     action fileformat_error {
         ErrorPolicy::handle_error(*this,
-            new FileformatError{n_lines, "The fileformat declaration is not 'fileformat=VCFv4.1'"});
+            new FileformatError{n_lines,
+                "The fileformat declaration is not valid (value must be one of 'VCFv4.1', 'VCFv4.2' or 'VCFv4.3')"});
```

Comment:
I thought this message was going to be reported in `extract_version`. This is misleading because it states non-valid fileformats are valid for this state machine.